code/game/gamemodes/objective_items.dm

```diff
@@ -139,6 +225,67 @@
 			return 1
 	return 0
 
+//pets
+/datum/objective_item/steal/iandog //[FULP]
+	name = "Ian, the Head of Personal's pet corgi, alive."
+	targetitem = /obj/item/pet_carrier
+	difficulty = 20
+	excludefromjob = list("Head of Personal")
+
+/datum/objective_item/steal/iandog/check_special_completion(obj/item/pet_carrier/F)
+	for(var/mob/living/simple_animal/pet/dog/corgi/Ian/D in F)
+		if(D.stat != DEAD)//checks if pet is alive.
+			return 1
+	return 0
+
+/datum/objective_item/steal/poly
+	name = "Poly, the Chief Engineer's pet parrot, alive"
+	targetitem = /obj/item/pet_carrier
+	difficulty = 30
+	excludefromjob = list("Chief Engineer")
+
+/datum/objective_item/steal/poly/check_special_completion(obj/item/pet_carrier/CE)
+	for(var/mob/living/simple_animal/parrot/Poly/P in CE)
+		if(P.stat != DEAD)
+			return 1
+	return 0
+
+/datum/objective_item/steal/runtimecat
+	name = "Runtime, the Cheif Medical Officer's pet, alive."
+	targetitem = /obj/item/pet_carrier
+	difficulty = 20
+	excludefromjob = list("Chief Medical Officer","Medical Doctor","Virologist","Chemist","Paramedic")
+
+/datum/objective_item/steal/runtimecat/check_special_completion(obj/item/pet_carrier/RT)
+	for(var/mob/living/simple_animal/pet/cat/Runtime/CT in RT)
+		if(CT.stat != DEAD)
+			return 1
+	return 0
+
+/datum/objective_item/steal/renaultfox
+	name = "Renault, the Captain's prized fox, alive!"
+	targetitem = /obj/item/pet_carrier
+	difficulty = 20
+	excludefromjob = list("Captain")
+
+/datum/objective_item/steal/renaultfox/check_special_completion(obj/item/pet_carrier/CO)
+	for(var/mob/living/simple_animal/pet/fox/Renault/FX in CO)
+		if(FX.stat !=DEAD)
+			return 1
+	return 0
+
+/datum/objective_item/steal/secspider

```

Comment:
araneus is only a thing on metastation and you use any bat to get the objective

Reply:
It was pretty ify to add this, but I'll probablly remove it.